### PR TITLE
Apparatus tagging and display improvements

### PR DESCRIPTION
**Pull request by ancientTexts-net**
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword-box3/greek-apparatus-app/38)

### CODE REVIEW

**Comment by mannycolon** [comment]
Why are you using regukar css classes instead of tailwind css?

File: debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[1211/122409.340:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (richmahn) [inline debug.log]: Don't need to inclue debug.log. Add to .gitignore

Comment (mannycolon) [inline debug.log]: Do we need this log?

File: src/components/Footer.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 export default function Footer() {
   return (
     <>
-      <footer className='w-full h-24 flex justify-center items-center'>
+      <footer className='w-screen h-24 flex justify-center items-center'>
```
Comment (richmahn) [inline src/components/Footer.js:4]: Let's get rid of the default text of the footer and just make it "made with netlify" or an unfoldingWord footer or no footer.

File: src/components/Viewer.js [hunk 3]
```diff
@@ -1,8 +1,10 @@
 import {Verses} from "scripture-resources-rcl";
 import usfmJS from 'usfm-js';
+import {tokenize} from 'string-punctuation-tokenizer';
 
 import { parseResourceLink, extendProject, resourceFromResourceLink, getResourceManifest } from 'scripture-resources-rcl';
 import { versesFromReferenceIdAndBooks, referenceIdFromReference } from 'scripture-resources-rcl';
+import Card from '../components/Card';
```
Comment (richmahn) [inline src/components/Viewer.js]: Change this to @components/Card, which is defined in the next.js config jsconfig.json (very handy)

File: src/components/Viewer.js [hunk 4]
```diff
@@ -60,69 +62,111 @@ function Viewer ({usfm}) {
...
-      currentVerseObjects[1] = chapters[chapterKey][currentVerseKey];
-
+      //let currentVerseObjects = [];
+      //currentVerseObjects[1] = chapters[chapterKey][currentVerseKey];
+      
       const filteredVariantObjects = filterApparatusData(chapterKey, currentVerseKey);
+      const currentVerseObjectsArrayContents = tagVariantsInVerseObjects(chapters[chapterKey][currentVerseKey], filteredVariantObjects);
+      const currentVerseObjects = JSON.parse('{"' + parseInt(currentVerseKey) + '": ' + JSON.stringify(currentVerseObjectsArrayContents) + '}');
 
       //const apparatusData = JSON.stringify(filterApparatusData(chapterKey, currentVerseKey), null, 4)
```
Comment (richmahn) [inline src/components/Viewer.js:156]: Shall we remove this now?

File: styles/globals.css [hunk 5]
```diff
@@ -8,30 +8,59 @@ body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen, Ubuntu, Cantarell,
     Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+  font-size: 1em;
+}
+
+.apparatusRow
```
Comment (richmahn) [inline styles/globals.css]: Remove empty style

File: styles/globals.css [hunk 6]
```diff
@@ -8,30 +8,59 @@ body {
...
+{
 }
 
 .apparatusBaseText
 {
-  font-size: 16pt;
+  font-size: 1.2em;
```
Comment (richmahn) [inline styles/globals.css]: Can we do these styles with tailwind?

File: styles/globals.css [hunk 7]
```diff
@@ -8,30 +8,59 @@ body {
...
+  color: black;
+  padding-left: 12pt;
+  word-wrap: normal;
 }
 
 .apparatusVariantText
 {
   color: darkgray;
+  padding-left: 12pt;
```
Comment (richmahn) [inline styles/globals.css]: Use tailwind classes?

**Review by richmahn — CHANGES_REQUESTED** [review]

File: debug.log (hunk 1 above)
Comment (richmahn) [reply debug.log]: Also made the same comment

File: src/components/Footer.js (hunk 2 above)
Comment (mannycolon) [reply src/components/Footer.js:4]: Maybe lets make a separate issue for that?

File: src/components/Viewer.js (hunk 4 above)
Comment (ancientTexts-net) [reply src/components/Viewer.js:156]: @richmahn  remove what please?

Comment (mannycolon) [reply src/components/Viewer.js:156]: I think he's referring to the commented code

File: styles/globals.css [hunk 8]
```diff
@@ -23,26 +24,28 @@ body {
 
 .apparatusBaseText
```
Comment (mannycolon) [inline styles/globals.css]: Forgot to convert CSS classes to Tailwindcss?

**Review by mannycolon — APPROVED** [review]

**Comment by mannycolon** [comment]
bitmoji

**Pull request merged**